portfolio/src/main/webapp/comments.js

```diff
@@ -207,3 +176,37 @@ function createNoCommentsTag() {
   noCommentsTag.innerText = 'No comments yet';
   document.getElementById('comments').appendChild(noCommentsTag);
 }
+
+// creates a button that acts like a link
+function createLinkButton(text, link) {
+  const button = document.createElement('button');
+  button.innerText = text;
+  button.onclick = function() {
+    window.location.href = link;
+  }
+  return button;
+}
+
+// creates a button that will delete given comment on click
+function createDeleteButton(comment) {
+  const deleteButton = document.createElement('button');
+  deleteButton.innerText = 'Delete';
+  deleteButton.onclick = function() {
+    const request = new Request('/delete-data?commentKey='+comment.key, {method : 'POST',});
+    fetch(request).then(() => {
+      window.location.reload(false);
+    })
+  }
+  return deleteButton;
+}
+
+function createShowReplyFormButton(comment) {
+  const showReplyFormButton = document.createElement('button');
+  showReplyFormButton.innerText = 'Reply';
+  showReplyFormButton.id = comment.key + '_button'; // this button must have unique id as I will change it's text
+                                                    // depending on form state
+  showReplyFormButton.onclick = function() {
+    toggleReplyField(comment.key);
+  };
+  return showReplyFormButton;
+}
```

Comment:
Please add a new line.

Reply:
oops, I was setting mac today, forgot to configure new lines, fixed it